### ISSUE TITLE
Focus-visible in Safari is not working

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1081]
## Description

The `focus-visible` polyfill we added in #724 is not working.

## Reproduction

1. Run the app and open http://localhost:8443/search/?q=cat in a desktop Safari browser
2. Tab through the search bar to the page menu and content switcher buttons.
3. See error: You are supposed to see pink ring around those buttons when you focus on them using <kbd>Tab</kbd>, but there is no indication of where the focus is.

## Expectation

Elements that have keyboard focus should have a pink ring around them.

## Environment

 - Device:  laptop
 - OS:  MacOS
 - Browser:  Safari

## Additional context

It is not working on staging and in production, too.
## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by sarayourfriend** [comment]
Relabeling as critical and a11y. @WordPress/openverse-frontend can someone with a mac pick up this issue?

**Comment by dhruvkb** [comment]
I can see the pink ring:
https://user-images.githubusercontent.com/16580576/159772928-099a321f-2afe-4592-aec3-08adc965d950.mov

**Comment by zackkrida** [comment]
I'm not able to see the focus rings, here's my Safari and computer information:

```
Safari: Version 15.3 (17612.4.9.1.8)
MacOS: 12.2.1 (21D62)
```

![CleanShot 2022-03-23 at 14 45 41@2x](https://user-images.githubusercontent.com/6351754/159773901-c5cf9943-202d-445d-aec2-a772bca75269.png)

**Comment by zackkrida** [comment]
As discussed elsewhere Safari `15.4` adds support for `focus-visible:`, causing this discrepancy. @obulat or @krysal as the only two remaining mac users will be able to debug this further?

**Comment by obulat** [comment]
Noting here that the `focus-visible` problem was re-introduced in the PR fixing the CSS styles import order, #993
